### PR TITLE
java objects

### PR DESCRIPTION
**Pull request by jeesupark96**
Screen Shot 2021-09-21 at 8 39 08 PM

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var person = {
+  firstName: "Jeesu",
+  lastName: "Park",
+  hobby: "basketball",
+};
+console.log(person);
+var firstName = person.firstName
```
Comment (scbowler) [inline prep-javascript-objects/main.js]: This should be `fullName` so both the first and last name together with a space between the names.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good,  just one update needed. Let me know if you have any questions.

File: prep-javascript-objects/main.js (hunk 1 above)
Comment (jeesupark96) [reply prep-javascript-objects/main.js]: oh I definitely messed that one up.. 
thank you haha

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**